luin/readability
# Title extraction from meta + regex fixes

## Changes:
- Update `getTitle` to check the meta tags for titles. (Twitter + Open Graph titles)
- Concatenate class and id with a newline. This allows you to match on things like `post` but only if it is at the start of the class/id.
- Update the regex to fix some sites that were failing to extract content.
- Add regression tests to check the extraction/title fixes.

## How to test:
- Run `npm test` with and without the helper changes to verify that they are fixing article extraction for the added tests.

Let me know if you have any questions/comments/concerns @luin :).
